frontend/src/components/UploadedImagesSection/UploadedImagesSection.js

```diff
@@ -0,0 +1,17 @@
+import UploadedImageBox from 'components/UploadedImage/UploadedImage';
+import { UploadedImagesSection } from 'components/UploadedImagesSection/UploadedImagesSection.styles';
+
+const imageURL1 = 'https://cataas.com/cat/says/wok';
+const imageURL2 = 'https://cataas.com/cat/says/h22';
+const imageURL3 = 'https://cataas.com/cat/says/hel';
+
+const ImagesSection = () => (
+  <UploadedImagesSection>
+    <UploadedImageBox url={imageURL1} />
```

Comment:
Consider using [keys](https://pl.reactjs.org/docs/lists-and-keys.html). I bet React already complains there are no keys used here.